implementation/test/TBTCSystemTest.js

```diff
@@ -50,6 +61,7 @@ describe("TBTCSystem", async function() {
 
       const result = await tbtcSystem.requestNewKeep.call(5, 10, 0, {
         value: openKeepFee,
+        from: accounts[0],
```

Comment:
Shouldn't it be called from `keepOwner`?

Reply:
it doesn't matter here, but can change for consistency